app/views/sequences/new.html.erb

```diff
@@ -10,6 +10,9 @@
 <%= render(partial: "observation_title",
            locals: { observation: @observation }) %>
 
-<%= form_for(:sequence, url: add_query_param(action: :create_sequence)) do |form| %>
-  <%= render(partial: "form_sequence", locals: { form: form }) %>
-<% end %>
+<%= render(
+  partial: "form",
+  locals: {
+    url: add_query_param(sequences_path(params: { obs_id: @observation.id }))
+  }
+) %>
```

Comment:
I agree.  This makes more sense than having the form outside the partial.  That was a bit weird.